docs/management/connectors/action-types/slack.asciidoc

```diff
@@ -69,16 +80,26 @@ Use the <<action-settings,Action configuration settings>> to customize connector
 
 [float]
 [[configuring-slack]]
-==== Configure a Slack account
+=== Configure a Slack account
 
-You need a https://api.slack.com/incoming-webhooks[Slack webhook URL] to 
-configure a Slack account. To create a webhook
-URL, set up an an **Incoming Webhook Integration** through the Slack console:
+Before you can create a Slack connector, you must configure your account and obtain the necessary URL or token.
+
+[float]
+[[configuring-slack-webhook]]
+==== Configure a Slack account with Webhook
 
 . Log in to http://slack.com[slack.com] as a team administrator.
-. Go to https://my.slack.com/services/new/incoming-webhook.
-. Select a default channel for the integration.
-+
-image::images/slack-add-webhook-integration.png[]
-. Click *Add Incoming Webhook Integration*.
+. Follow first 3 paragraphs of https://api.slack.com/messaging/webhooks.
 . Copy the generated webhook URL so you can paste it into your Slack connector form.
+. If you are using the <<action-settings,`xpack.actions.allowedHosts`>> setting, make sure the hostname from the URL is added to the allowed hosts.
```

Comment:
> . If you are using the <<action-settings,`xpack.actions.allowedHosts`>> setting, make sure the hostname from the URL is added to the allowed hosts.

This was originally elsewhere in the page, but IMO it seems most pertinent here.